projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.module.ts

```diff
@@ -8,10 +8,9 @@ import { TooltipService } from './tooltip.service';
 import { InjectionService } from './injection.service';
 
 @NgModule({
-  declarations: [TooltipContentComponent, TooltipDirective],
-  providers: [InjectionService, TooltipService],
-  exports: [TooltipContentComponent, TooltipDirective],
-  imports: [CommonModule],
-  entryComponents: [TooltipContentComponent]
```

Comment:
Is there a specific reason this was removed?

Reply:
i think the CLI update command removed it automatically 🤔

Reply:
Ahh seems like angular 13 deprecated `entryComponents` and it's no longer needed

Reply:
We might need to keep it for now... see the note "You may still need to keep these if building a library that will be consumed by a View Engine application."